scripts/basic-workflow-tests.sh

```diff
@@ -34,12 +34,14 @@ task() {
 }
 
 set_defaults() {
+    GENERATE_ARGS+=("--num-cpus" "$(nproc)")
```

Comment:
At this point, I would initialize the array directly with those args since they get set anyways.

Reply:
good suggestion - done!